raiden/tests/utils/smoketest.py

```diff
@@ -122,32 +80,30 @@ def run_restapi_smoketests():
     assert response_json[0]['balance'] > 0
 
 
-def run_smoketests(raiden_service: RaidenService, test_config: Dict, debug: bool = False):
+def run_smoketests(
+        raiden_service: RaidenService,
+        transport: str,
+        token_addresses,
+        discovery_address,
+        debug: bool = False,
+):
     """ Test that the assembled raiden_service correctly reflects the configuration from the
     smoketest_genesis. """
     try:
         chain = raiden_service.chain
-        assert (
```

Comment:
these asserts don't make sense since the smart contracts are redeployed and the registries are instantiated with these new addresses.